torch/_inductor/triton_helpers.py

```diff
@@ -1,6 +1,15 @@
 import triton
 import triton.language as tl
 
+TRITON_HAS_REDUCE = False  # hasattr(tl, "reduce")
```

Comment:
This gets uncommented in #100569, but it's good to have CI run on the alternate implementation first.